docs/md/workload.md

```diff
@@ -0,0 +1,56 @@
+# Workloads
+
+## What is it?
+Within the context of Workflow Launcher, a workload is a discrete body of work, which takes data from a source, pushes it into a workflow executor for analysis, and then delivers the results of the analysis to an output location (also known as a sink).
+
+## Workload Components
+### Source
+### Executor
+### Sink
+
+## Example workload 
+(The specific values below are from the COVID-19 Surveillance in Terra project. Workloads for other projects may have different fields available in the source, executor and sink).
+
+```
+{
+    "watchers": [],
```

Comment:
Would be good to have a concrete example here rather than the empty list.

Reply:
Done